src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/EventController.java

```diff
@@ -0,0 +1,103 @@
+    EventDateTime end = new EventDateTime().setDateTime(endDateTime).setTimeZone("America/Los_Angeles");
+
+    @GetMapping("/insert")
+    public String insertEvent() throws IOException {
+
+        //Google Event
+        com.google.api.services.calendar.Calendar service = getCalendarService();
+        Event event = service.events().insert("primary", new Event()
+                .setDescription("New Event")
+                .setSummary("Barbi is testing the database")
+                .setStart(start)
+                .setEnd(end))
+                .execute();
+
+        //Google Calendar
+        Calendar googleCalendar = service.calendars().get("primary").execute();
+
+        //Opal Calendar
+        CalendarModel calendar = calendarModelService.setCalendarAttributes(googleCalendar);
+
+        //Opal Event
+        EventModel eventModel = eventModelService.setAttributes(event, googleCalendar);
+
+        // saving to the database
+        eventModelService.saveEvent(eventModel);
+
+        return "redirect:https://calendar.google.com/calendar/b/2/r";
+
+    }
+
+    @GetMapping("/update")
+    public String updateEvent() throws IOException {
+        com.google.api.services.calendar.Calendar service =
+                getCalendarService();
+        Event event = service.events().get("primary", "7nhdmehd85ogf46u0sdcpmfq7h").execute();
+        event.setSummary("Trolloloooo!!!");
+
+        Event updatedEvent = service.events().update("primary", event.getId(), event).execute();
+        return "redirect:https://calendar.google.com/calendar/b/1/r";
+    }
+
+    @GetMapping("/events/list")
+    public String listEvents() throws IOException {
+        com.google.api.services.calendar.Calendar service = getCalendarService();
+        String pageToken = null;
+        do {
+            Events events = service.events().list("primary").setPageToken(pageToken).execute();
+            List<Event> items = events.getItems();
+            for (Event event : items) {
+                System.out.println(event.getSummary());
+            }
+            pageToken = events.getNextPageToken();
+        } while (pageToken != null);
+        return "redirect:https://calendar.google.com/calendar/b/1/r";
+    }
+  
+    @GetMapping("/event/find")
+    public String getEvent() throws IOException {
+        com.google.api.services.calendar.Calendar service = getCalendarService();
+        Event event = service.events().get("primary", "7nhdmehd85ogf46u0sdcpmfq7h").execute();
+        System.out.println(event.getSummary());
+        return "redirect:https://calendar.google.com/calendar/b/1/r";
+    }
+  
+    @DeleteMapping("/event/delete")
+    public String deleteEvent() throws IOException {
+        com.google.api.services.calendar.Calendar service = getCalendarService();
+        service.events().delete("primary", "eventId").execute();
+        return "redirect:https://calendar.google.com/calendar/b/1/r";
+    }
```

Comment:
Endpoints are not ok, we have "/event" on the class then on some of the methods too.